manifests/g/GitTools/GitVersion/5.11.1/GitTools.GitVersion.installer.yaml

```diff
@@ -0,0 +1,36 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS2.CRLF.7-3-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: GitTools.GitVersion
+PackageVersion: 5.11.1
+InstallerType: zip
+Commands:
+- gitversion
+ReleaseDate: 2022-11-08
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Git.Git
+Installers:
```

Comment:
```suggestion
    - PackageIdentifier: Git.Git
Installers:
```

Reply:
It appears that I added an incorrect line ending, or other invisible character that's blocking dependencies. I believe that I have removed it with this suggestion.

Reply:
It seems the Commit suggestion for some reason is disabled, so what line ending should be? I will apply it myself and push again